packages/cdktf-cli/lib/environment.ts

```diff
@@ -0,0 +1,15 @@
+// All switches based on environment variables in one place
+
+function envVarToBoolean(envVar: string | undefined): boolean {
+  return envVar === "true" || envVar === "1";
+}
+
+export const DISABLE_VERSION_CHECK = envVarToBoolean(
+  process.env.DISABLE_VERSION_CHECK
+);
+export const CHECKPOINT_DISABLE = envVarToBoolean(
+  process.env.CHECKPOINT_DISABLE
```

Comment:
We should not do this for this env var.
> Checkpoint itself can be entirely disabled for all HashiCorp products by setting the environment variable CHECKPOINT_DISABLE to any non-empty value.

https://www.terraform.io/cli/commands#upgrade-and-security-bulletin-checks

Let's allow any value and add a note why.